test/metabase/driver/postgres_test.clj

```diff
@@ -1089,21 +1104,22 @@
 
 (deftest pkcs-12-extension-test
   (testing "Uploaded PKCS-12 SSL keys are stored in a file with the .p12 extension (#20319)"
-    (is (true?
```

Comment:
I shuffled this whole thing around to get it to shut up about a reflection warning calling `.getAbsolutePath` on an unknown type